apps/validate/validate.py

```diff
@@ -55,6 +55,8 @@ def get_validator_schema(schema):
     validator_schema = {key: val for key, val in schema.items() if val is not None}
 
     if validator_schema.get('required') and not validator_schema.get('minlength'):
+        if 'mandatory_in_list' in validator_schema and 'subject' in validator_schema['mandatory_in_list']['scheme']:
+            validator_schema['schema']['schema']['scheme']['allowed'] = ['subject_custom', 'category']
```

Comment:
subject_custom and category are the names of a custom fields, so they cannot be used as constants

Reply:
I am unable to edit the content profiles. So, I think t depends on this https://dev.sourcefabric.org/browse/SDNTB-550.